packages/app/docs/stories/appConfirmModal.story.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { Button } from 'react-rainbow-components';
+// import styled from 'styled-components';
+// import { RainbowLogo } from '@rainbow-modules/icons';
+import RainbowFirebaseApp from '../../src/components/App';
+import ConfirmModal from '../../src/components/ConfirmModal';
+import { confirmModal } from '../../src/actions';
+
+const DeleteIcon = () => (
```

Comment:
@TahimiLeonBravo it makes sense to move this icon into @rainbow-modules/icons??

Reply:
as i wasn't sure where to put the icon as it is used only in the example...

Reply:
yes, let's move it to the icons module